migrations_local/versions/11bef167c5c2_add_all_current_tables.py

```diff
@@ -174,113 +212,220 @@ def upgrade():
         sa.Column("effect_choice_uuid", mssql.UNIQUEIDENTIFIER(), nullable=True),
         sa.Column("solution_choice_uuid", mssql.UNIQUEIDENTIFIER(), nullable=True),
         sa.Column("consent", sa.Boolean(), nullable=True),
+        # sa.ForeignKeyConstraint(
+        #     ["alignment_feed_uuid"],
+        #     ["alignment_feed.alignment_feed_uuid"],
+        # ),
+        # sa.ForeignKeyConstraint(
+        #     ["alignment_scores_uuid"],
+        #     ["alignment_scores.alignment_scores_uuid"],
+        # ),
+        # sa.ForeignKeyConstraint(
+        #     ["conversation_uuid"],
+        #     ["conversations.conversation_uuid"],
+        # ),
+        # sa.ForeignKeyConstraint(
+        #     ["effect_choice_uuid"],
+        #     ["effect_choice.effect_choice_uuid"],
+        # ),
+        # sa.ForeignKeyConstraint(
+        #     ["quiz_uuid"],
+        #     ["scores.quiz_uuid"],
+        # ),
+        # sa.ForeignKeyConstraint(
+        #     ["solution_choice_uuid"],
+        #     ["solution_choice.solution_choice_uuid"],
+        # ),
         sa.PrimaryKeyConstraint("conversation_uuid"),
     )
+
+    op.create_foreign_key(
```

Comment:
I moved this part to be here instead of down below like it was in the previous migration script.